src/index.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpFunction } from '@google-cloud/functions-framework';
+import { Log, Logging } from '@google-cloud/logging';
+import { SUCCESS_MESSAGE } from './helpers/constants';
+import { writeLog } from './helpers/logs';
+import { handleSlackMessageEvent } from './helpers/slack';
+const { WebClient } = require('@slack/web-api');
+const fs = require('fs');
+
+// Creates a client
+const logging: Logging = new Logging();
+
+// Selects the log to write to
+const log: Log = logging.log(process.env.LOG_NAME || 'xplorers-bot-log');
+
+const slackWebClient: typeof WebClient = new WebClient(process.env.SLACK_OAUTH_TOKEN || fs.readFileSync('/etc/secrets/slack-oauth-token'))
```

Comment:
I can see `typeof WebClient` being written in many places. Do you think it would be better to create a separate type for this?

Reply:
Created a new type for slackWebClient 👍 v